docs/plugins.rst

```diff
@@ -390,79 +390,3 @@ R
-  and validators for several types. This plugin requires that you have the VTK
-  Python package exposed in Girder Worker's Python environment. The ``geometry`` type
-  represents 3D geometry.
-* **Converters added:**
-    * ``geometry/vtkpolydata`` |ba| ``geometry/vtkpolydata.serialized``
-    * ``table/rows`` |ba| ``table/vtktable``
-    * ``table/vtktable`` |ba| ``table/vtktable.serialized``
-    * ``tree/nested`` |ba| ``tree/vtktree``
-    * ``tree/vtktree`` |ra| ``tree/newick``
-    * ``tree/vtktree`` |ba| ``tree/vtktree.serialized``
-    * ``graph/networkx`` |ba| ``graph/vtkgraph``
-    * ``graph/vtkgraph`` |ba| ``graph/vtkgraph.serialized``
-
-* **Validators added:**
-    * ``geometry/vtkpolydata``: A vtkPolyData_ object.
-    * ``geometry/vtkpolydata.serialized``: A vtkPolyData serialized with vtkPolyDataWriter_.
-    * ``table/vtktable``: A vtkTable_.
-    * ``table/vtktable.serialized``: A vtkTable serialized with vtkTableWriter_.
-    * ``tree/vtktree``: A vtkTree_.
-    * ``tree/vtktree.serialized``: A vtkTree serialized with vtkTreeWriter_.
-    * ``graph/vtkgraph``: A vtkGraph_.
-    * ``graph/vtkgraph.serialized``: A vtkGraph serialized with vtkGraphWriter_.
-
-.. note :: vtkGraphs lose their actual node values as they are represented by their index.
-  In addition, nodes and edges are given all metadata attributes with defaults if they do not specify the metadatum themselves.
-  This is noted further in :py:mod:`girder_worker.core.plugins.vtk.converters.graph.networkx_to_vtkgraph`
-
-.. _vtkGraph: http://www.vtk.org/doc/nightly/html/classvtkGraph.html
-.. _vtkGraphWriter: http://www.vtk.org/doc/nightly/html/classvtkGraphWriter.html
-.. _vtkTree: http://www.vtk.org/doc/nightly/html/classvtkTree.html
-.. _vtkTreeWriter: http://www.vtk.org/doc/nightly/html/classvtkTreeWriter.html
-.. _vtkTable: http://www.vtk.org/doc/nightly/html/classvtkTable.html
-.. _vtkTableWriter: http://www.vtk.org/doc/nightly/html/classvtkTableWriter.html
-.. _vtkPolyData: http://www.vtk.org/doc/nightly/html/classvtkPolyData.html
-.. _vtkPolyDataWriter: http://www.vtk.org/doc/nightly/html/classvtkPolyDataWriter.html
-.. _vtkTree: http://www.vtk.org/doc/nightly/html/classvtkTree.html
-
-Types
------
-
-* **Plugin ID:** ``types``
-* **Description:** This plugin allows type and format annotations of inputs and outputs
-  to be added to task specs and IO bindings. It can also perform automatic conversion between
-  different formats for known types, as well as validating the correctness of the data formats.
-  These behaviors are enabled with optional boolean arguments to the ``run`` task: ``validate``
-  and ``auto_convert``. Other plugins such as VTK and R add additional types and formats to
-  the typesystem supported by this plugin. The full documentation including a list of supported
-  types and formats can be found in the :ref:`types-and-formats` section.
-
-.. |ra| unicode:: 8594 .. right arrow
-.. |ba| unicode:: 8596 .. bidirectional arrow
```

Comment:
The above two lines need to remain, they are used in other places in the docs that were not deleted.